docs/how-to-guides/isis-developer-guides/building-and-contributing-to-isis-tutorial.md

```diff
@@ -0,0 +1,87 @@
+
+A great way to get a grasp on the converting process is by looking at previous examples so here are some PRs:
+
+- [`gaussstretch` PR #5259](https://github.com/DOI-USGS/ISIS3/pull/5259)
+- [`skypt` PR #5444](https://github.com/DOI-USGS/ISIS3/pull/5444)
+  
+## 3. Run Tests
+
+Check out [Running Tests](https://astrogeology.usgs.gov/docs/how-to-guides/isis-developer-guides/developing-isis3-with-cmake/#running-tests) to get an idea on how to run your tests.
+
+As an example, let's look at the [`skypt` test suite](https://github.com/DOI-USGS/ISIS3/blob/dev/isis/tests/FunctionalTestsSkypt.cpp). You can run the all of `skypt`'s tests with `ctest -R skypt`. Let's say you want to test only the first test case in *FunctionalTestsSkypt.cpp*, run `ctest -R FunctionalTestSkyptDefault`. 
+
+## 4. Update CHANGELOG
+
+Make sure you update the *CHANGELOG.md* located in the root of the `ISIS3` folder by adding an entry under the **Unreleased** heading and **Changed** subheading.
+
+## 5. Create PR
+
+Once your tests are in good shape with passing marks, let's create a Pull Request (PR). Go to your forked ISIS3 repo page and select the **Pull requests** tab towards the top of the page. Then, hit the green **New pull request** button which will take you to a page where you select your base and head repositories to compare before creating the pull request. The base repository is the repo you will be pushing your changes *to*, so it should be set to 
+```
+base repository: **DOI-USGS/ISIS3**, base: **dev**
+```
+
+The head repository is where the changes will be coming *from*, a.k.a. where you have been pushing your local changes to:
+```
+head repository: **<your-account-name>/ISIS3**, compare: **<branch-name>**
+```
+
+Check the changes on the comparison page then hit the green **Create pull request** button. You will have the opportunity to make more changes to your branch after creating the PR so no worries if you are unable to catch something in this stage. 
+
+Now, you should be on a page titled **Open a pull request**, the step to add further details regarding your PR. Under **Add a title**, title your PR as *APP-NAME app and tests refactor* (e.g., *gaussstretch app and tests refactor*).
+
+In the body of the PR, there should already be a PR template ready for you to fill out, you can check out the preview of what the PR would look like when created when you select the **Preview** tab. Here's a general outline in what the PR should look like:
+
+```md
+**Description**
+Describe what app, tests, and any appropriate files have been affected. 
+You can also note any weird quirks or changes to the main code that seemed necessary to give the reviewer more context.
+
+**Related Issue**
+Usually, you would add here *Addresses <issue-url>*. 
+If there is no existing GitHub issue to link, you can leave this blank.
```

Comment:
I thought I remembered a policy of "each change has to have an associated issue," but I may be misremembering?  I'm not able to find any documentation to support that claim.  @Kelvinrr @acpaquette @amystamile-usgs do you have any recollection of that type of policy in the past?

Reply:
Contributing docs does not say this explicitly. So I think it's fine.